### PR TITLE
Fix make doesn't build src files and build error

### PR DESCRIPTION
**Pull request by jin-tron**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,3 +13,7 @@ lintf:
 build:
 	prototool all
 	go mod tidy
+	for dir in $(SRC_DIR); \
+	do \
+	eval go build $$dir; \
```
Comment (taiyangc) [inline Makefile]: does it work with just `go build $(dir)`?

Comment (taiyangc) [reply Makefile]: waiting for a reason on eval, normally it's unsafe to use

Comment (jin-tron) [reply Makefile]: yes, without eval it also works, let's me remove this eval

File: Makefile [hunk 2]
```diff
@@ -11,3 +13,7 @@ lintf:
 build:
 	prototool all
 	go mod tidy
+	for dir in $(SRC_DIR); \
+	do \
+	go build $$dir; \
+	done
```
Comment (wxue) [inline Makefile]: ```go build ./...```
should be all you need here

Comment (jin-tron) [reply Makefile]: will update in next PR coming soon

Comment (wxue) [reply Makefile]: ok, sounds good

**Pull request merged**